glean-core/python/setup.py

```diff
@@ -48,9 +55,12 @@
 
 setup_requirements = []
 
-shared_object_build_dir = "../../target/debug/"
-if os.environ.get("GLEAN_PYTHON_MINGW_BUILD"):
+if mingw_arch == "i686":
+    shared_object_build_dir = "../../target/i686-pc-windows-gnu/debug/"
+elif mingw_arch == "x86_64":
     shared_object_build_dir = "../../target/x86_64-pc-windows-gnu/debug/"
+else:
+    shared_object_build_dir = "../../target/debug/"
```

Comment:
I did not realize we hard-code the output directory.
This is ... not optimal I assume.

1. We probably want release builds eventually
2. The sub directory of the target is only like that if cross-compiled.

We should file a bug and fix this up later.

Reply:
Yeah -- we've had a bug for this for a while: https://bugzilla.mozilla.org/show_bug.cgi?id=1597714

The cross compiling subdirectories are only used when cross compiling is indicated using the `GLEAN_PYTHON_` env variables, and that's really just for making the official builds in CircleCI.  It's not really "supported" for developer use, as indicated by the docs:  https://mozilla.github.io/glean/book/dev/python/setting-up-python-build-environment.html#building-wheels-for-windows

Reply:
👍 so that's how few things I know about our Python package :D